04 Strategy Library/88 52-Weeks High Effect/01 Introduction.html

```diff
@@ -0,0 +1,15 @@
+<p>
+    The “52-week high effect” states that stocks with prices close to the 52-week highs have better subsequent returns than stocks with prices far from the 52-week highs. 
+    Investors use the 52-week high as an “anchor” which they value stocks against. 
```

Comment:
This sentence doesn't make sense by itself, please combine with the sentence below